src/platform/Linux/NFCCommissioningManagerImpl.cpp

```diff
@@ -535,6 +526,21 @@ void NFCCommissioningManagerImpl::_Shutdown()
 
 // ===== start implement virtual methods on NfcApplicationDelegate.
 
+CHIP_ERROR NFCCommissioningManagerImpl::StartNFCProcessingThread()
+{
+    // Creates an Application Context to the PC/SC Resource Manager.
+    long result = SCardEstablishContext(SCARD_SCOPE_SYSTEM, NULL, NULL, &hPcscContext);
+    CHECK("SCardEstablishContext", result)
```

Comment:
nit (maybe for the future): rename `CHECK` to `CHECK_FOR_SCARD_SUCCESS` or `VerifyScardSuccessOrReturnFailure` something like that ... it was not immediately obvious to me what it does.